frontend/src/common.js

```diff
@@ -209,7 +210,7 @@ export async function getSource(file) {
   const response = await fetch(
     `https://hg.mozilla.org/mozilla-central/raw-file/tip/${file}`
   );
-  return await response.text();
+  return response.text();
```

Comment:
Don't we need to await here?

Reply:
Come on Marco, https://eslint.org/docs/rules/no-return-await


I learned that today.